src/components/PopupWithForm.js

```diff
@@ -16,15 +17,19 @@ export default class PopupWithForm extends Popup {
     return formValues;
   }
 
-  // closePopup() {
-  //   super.closePopup();
-  // }
+ openPopup() {
+  this._saveButton.textContent = "Save";
+  super.openPopup();
+ }
```

Comment:
**Could be improved**

You can make ` setInputValues` in class `PopupWithForm`, which can insert data into inputs:

```js

  setInputValues(data) {
    this._inputList.forEach((input) => {
      // Here you insert the `value` by the `name` of the input
      input.value = data[input.name];
    });
  }



```

So, this way you’ll not have to search the inputs of the profile in `index.js`